tests/sync/test_ongoing_chat.py

```
@@ -0,0 +1,228 @@
+"""Test the ongoing chat functionality"""
+
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)